src/CommunityToolkit.Maui/Behaviors/ICommunityToolkitBehavior.shared.cs

```diff
@@ -89,7 +89,7 @@ protected void OnViewPropertyChanged(object? sender, PropertyChangedEventArgs e)
 		}
 		catch (Exception ex) when (Options.ShouldSuppressExceptionsInBehaviors)
 		{
-			Trace.WriteLine(ex);
+			Trace.TraceError("{0}", ex);
```

Comment:
Please do not use `TraceError` in the library. It causes crash reporting tools like sentry.io to report it as an unhandled exception. Leading to our library causing erroneous bug reports for our users.